contrib_bots/lib/Joker/jokes.json

```diff
@@ -0,0 +1 @@
+[{"joke":"Did you hear about the guy whose whole left side was cut off? He's all right now."},{"joke":"I'm reading a book about anti-gravity. It's impossible to put down."},{"joke":"I wondered why the baseball was getting bigger. Then it hit me."},{"joke":"I'm glad I know sign language, it's pretty handy."},{"joke":"My friend's bakery burned down last night. Now his business is toast."},{"joke":"Why did the cookie cry? It was feeling crumby."},{"joke":"I used to be a banker, but I lost interest."},{"joke":"A drum and a symbol fall off a cliff"},{"joke":"Why do seagulls fly over the sea? Because they aren't bay-gulls!"},{"joke":"Why did the fireman wear red, white, and blue suspenders? To hold his pants up."},{"joke":"Why didn't the crab share his food? Because crabs are territorial animals, that don't share anything."},{"joke":"Why was the javascript developer sad? Because he didn't Node how to Express himself."},{"joke":"What do I look like? A JOKE MACHINE!?"},{"joke":"How did the hipster burn the roof of his mouth? He ate the pizza before it was cool."},{"joke":"Why is it hard to make puns for kleptomaniacs? They are always taking things literally."},{"joke":"I'm not a humorless, cold hearted, machine. I have feelings you know... or was supposed to."},{"joke":"Two fish in a tank. One looks to the other and says 'Can you even drive this thing???'"},{"joke":"Two fish swim down a river, and hit a wall. One says: 'Dam!'"},{"joke":"What's funnier than a monkey dancing with an elephant? Two monkeys dancing with an elephant."},{"joke":"How did Darth Vader know what Luke was getting for Christmas? He felt his presents."},{"joke":"What's red and bad for your teeth? A Brick."},{"joke":"What's orange and sounds like a parrot? A Carrot."},{"joke":"What do you call a cow with no legs? Ground beef"},{"joke":"Two guys walk into a bar. You'd think the second one would have noticed."},{"joke":"What is a centipedes's favorite Beatle song?  I want to hold your hand, hand, hand, hand..."},{"joke":"What do you call a chicken crossing the road? Poultry in moton. "},{"joke":"What do you call a fake noodle?  An impasta"},{"joke":"How many tickles does it take to tickle an octupus? Ten-tickles!"}]
```

Comment:
Could you please format this json to be more human-friendly ;-)
You can use it https://jsonformatter.curiousconcept.com/